src/Wt/Dbo/FixedSqlConnectionPool.C

```diff
@@ -39,6 +39,7 @@ FixedSqlConnectionPool::FixedSqlConnectionPool(std::unique_ptr<SqlConnection> co
 {
   SqlConnection *conn = connection.get();
   impl_->freeList.push_back(std::move(connection));
+  impl_->timeout = std::chrono::steady_clock::duration::zero();
```

Comment:
I think it's preferable to default-initialize the `timeout`, like:

```cpp
std::chrono::steady_clock::duration timeout{};
```

Reply:
I'd usually agree, but in this case I decided to go with the `zero` initialization since you explicitly compare against `zero` in `getConnection` which implied to me that you expect the default value to be `zero`, and not whatever aggregate initialization delivers (though it should be 0 too). I took a quick peek in Visual Studio's STL implementation and they explicitly initialize the `zero` function return value with 0, so even they don't rely on aggregate initialization for the `zero` duration.

I can of course change it if you want. Just let me know. In the end, what really matters is that the issue gets fixed. Otherwise I might offer to reword the commit message to make clear that it is a zero and not a default initialization, to ease possible confusion.

Reply:
I consulted the standards document and now I'm indeed doubting whether it is indeed mandated that `std::chrono::steady_clock::duration::zero()` is the same as `std::chrono::steady_clock::duration{}`.

I guess to be completely sure it's fine we should use `zero()`. I'd still move it to the declaration of the `struct` rather than here, though. In a lot of cases in Wt we can't do it because our Wt-to-JWt toolchain doesn't support it, but it's fine in Wt::Dbo.